psycop/projects/cvd/model_training/data_loader/trainval_loader.py

```diff
@@ -0,0 +1,40 @@
+from collections.abc import Sequence
+from pathlib import Path
+
+import polars as pl
+from functionalpy import Seq
+
+from psycop.common.model_training_v2.config.baseline_registry import BaselineRegistry
+from psycop.common.model_training_v2.trainer.base_dataloader import BaselineDataLoader
+
+
+class MissingPathError(Exception):
+    ...
+
+
+@BaselineRegistry.data.register("parquet_vertical_concatenator")
+class ParquetVerticalConcatenator(BaselineDataLoader):
```

Comment:
I love this way of doing it -- removes a lot of complexity from the data loaders. Strong vote for adding this to common

Reply:
(code is nice and simple and naming is good, ok for moving this to main as is from me)